src/liquid/components/ld-link/test/ld-link.e2e.ts

```diff
@@ -0,0 +1,169 @@
+import {
+  analyzeAccessibility,
+  getPageWithContent,
+} from '../../../utils/e2e-tests'
+import { LdIcon } from '../../ld-icon/ld-icon'
+import { LdLink } from '../ld-link'
+
+describe('ld-link', () => {
+  describe('web component', () => {
+    it('basic', async () => {
+      const page = await getPageWithContent('<ld-link>Link</ld-link>')
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot()
+
+      const accessibilityReport = await analyzeAccessibility(page)
+      expect(accessibilityReport).toHaveNoAccessibilityIssues()
+    })
+
+    it('small', async () => {
+      const page = await getPageWithContent('<ld-link size="sm">Link</ld-link>')
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot()
+
+      const accessibilityReport = await analyzeAccessibility(page)
+      expect(accessibilityReport).toHaveNoAccessibilityIssues()
+    })
+
+    it('large', async () => {
+      const page = await getPageWithContent('<ld-link size="lg">Link</ld-link>')
+      const results = await page.compareScreenshot()
+      expect(results).toMatchScreenshot()
+
+      const accessibilityReport = await analyzeAccessibility(page)
+      expect(accessibilityReport).toHaveNoAccessibilityIssues()
+    })
+
+    // TODO: Element has insufficient color contrast
```

Comment:
No problem. On disabled elements the contrast ratio is not really an issue. So you can remove the a11y test here. Actually you don't need to run a11y tests in each test, since this is unnecessary repetition. If you want to catch the worst scenario, run an a11y test on the default state of a link inside an ld-typo component with variant body-s (the link should inherit the font size) and theme tea.